{
  Throwable t=new Throwable();
  mockCollaborator.oneThrowable(t);
  mockControl.replay();
  try {
    this.testBean.exceptional(t);
    fail("should throw exception");
  }
 catch (  Throwable x) {
  }
  mockControl.verify();
}
